haiku-apps/alivejournal/alivejournal-1.61.recipe

```diff
@@ -0,0 +1,50 @@
+SUMMARY="Client for LiveJournal online journalling community"
+DESCRIPTION="AliveJournal is a revamped version of Simon Huet's original client."
+HOMEPAGE="https://github.com/grahams/alivejournal"
+COPYRIGHT="
+	2000 Simon Huet
+	2010 Sean Graham
```

Comment:
I'm so sorry, the style I used for the COPYRIGHT variable in my previous review was wrong with regards to the tabs and newlines. So you will have to do a second small edit here.
```diff
-COPYRIGHT="
-	2000 Simon Huet
+COPYRIGHT="2000 Simon Huet
-	2010 Sean Graham
-	"
+	2010 Sean Graham"
```

(I had in mind the style for PROVIDES, REQUIRES, etc..., but the style for COPYRIGHT is different.)

Reply:
Oh, it's fine. I make mistakes all the time :)